validate-samples.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+BLUE='\033[1;34m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+BOLD='\033[1m'
+
+if ! command -v jq &> /dev/null
+then
+  echo
+  echo "#### ERROR ####"
+  echo "####"
+  echo "#### Please install the 'jq' tool before being able to use this script"
+  echo "#### see https://stedolan.github.io/jq/download"
+  echo "####"
+  echo "###############"
+  exit 1
+fi
+
+if ! command -v jsonschema-cli &> /dev/null
+then
+  echo
+  echo "#### ERROR ####"
+  echo "####"
+  echo "#### Please install the 'jsonschema-cli' tool before being able to use this script"
+  echo "#### see https://pypi.org/project/jsonschema-cli/"
+  echo "####"
+  echo "###############"
+  exit 1
+fi
+
+BASE_DIR=$(cd "$(dirname "$0")" && pwd)
+
+rm -r validate-output.txt &> /dev/null
+
+for schema in "devfile" "devworkspace" "devworkspace-template"
+do
+  schemaPath="./schemas/${schema}.json"
+  devfiles=$(jq -r ".\"yaml.schemas\".\"${schemaPath}\"[]" .theia/settings.json)
```

Comment:
This line is hard to read, can it be simplified?
```bash
devfiles=$(jq -r '."yaml.schemas"."${schemaPath}"[]' --arg schemaPath=$schemaPath .theia/settings.json)
```
or similar?

Reply:
The proposed change didn't seem to work (using an arg as an object key isn't straightforward).
But I still simplified the expression and removed the `\"` parts which made it hard to read.